google-ads-examples/src/main/java/com/google/ads/googleads/examples/utils/ArgumentNames.java

```diff
@@ -38,4 +38,6 @@
   public static final String REPLACE_EXISTING_TREE = "--replaceExistingTree";
   public static final String PAGE_URL = "--pageUrl";
   public static final String LABEL_ID = "--labelId";
+  public static final String FEED_ID = "--feedId";
+  public static final String FEED_ITEM_ID = "--feedItemId";
```

Comment:
Can you put these in roughly sorted order please - I know the rest of the file is a mess, but maybe before KEYWORD_PLAN_ID is reasonable.